src/parser.ts

```diff
@@ -121,11 +121,13 @@ function parseNonLiteral(
 ): AST {
   const definitions = getDefinitionsMemoized(getRootSchema(schema as any)) // TODO
   const keyNameFromDefinition = findKey(definitions, _ => _ === schema)
+  const jsdoc = parseTags(schema, options.jsdocTags);
+  const jsdocOrComment = isEmpty(jsdoc) ? { comment: schema.description } : { jsdoc: jsdoc };
```

Comment:
What happens when a schema has both unsupported properties, and a description? (hint: we should emit both)